portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,27 +27,40 @@
 @WebServlet("/comment")
 public class DataServlet extends HttpServlet {
 
+  private final String HTML_Contant_Type = "text/html";
```

Comment:
Constants should be all upper case. Also fix the typo.

Reply:
ahhh just fixed it.